src/window.rs

```diff
@@ -123,24 +129,6 @@ impl From<u64> for WindowId {
     }
 }
 
```

Comment:
Let's add a deprecated alias here.

```suggestion

#[deprecated = "renamed to `WindowAttributes`"]
pub type WindowBuilder = WindowAttributes;

```

Reply:
True, it's not just a rename, though it does provide much the same functionality, so having an alias is more discoverable for people that don't read the changelog...

But not a hill I want to die on.

Reply:
I mean, you still link to them from the changelog, I just want this thing to be present in rustdoc as well, since the changelog itself has a lot of things.

Reply:
Well, there's `#[doc(hidden)]` for that. So maybe:

```suggestion

#[deprecated = "functionality moved to `WindowAttributes`"]
#[doc(hidden)]
pub type WindowBuilder = WindowAttributes;

```

Reply:
Hm, I commented in the wrong thread, but I don't think it really matters.

The thing is that we removed the _core_ method, so you kind of discover the `WindowAttributes` one way or another.

Reply:
Fair enough, I see that it doesn't matter much.